src/Movie.elm

```diff
@@ -77,27 +133,91 @@ movieCard selectedGenres movie =
                 _ ->
                     Set.size (Set.intersect movie.genres selectedGenres) == 0
     in
-        a
+        button
             [ classList
                 [ ( Style.MovieCard, True )
                 , ( Style.Filterable, True )
                 , ( Style.Filtered, filtered )
                 ]
-            , href movie.url
-            , target "_blank"
+            , onClick <| focusMovie <| Selected movie
+            , type_ "button"
             ]
-            [ img
-                [ class [ Style.Poster ]
-                , src ("posters/" ++ movie.img)
-                ]
-                []
+            [ moviePoster movie
             , div
                 [ class [ Style.Title ] ]
                 [ text movie.title ]
             , notesView movie
             ]
 
 
+ratingsList : List Rating -> Html msg
+ratingsList ratings =
+    ratings
+        |> List.map (\l -> li [] [ text (l.source ++ " - "), b [] [ text l.value ] ])
+        |> ul []
+
+
+movieModalBase : (MovieSelection -> msg) -> List (Html msg) -> Html msg
+movieModalBase focusMovie contents =
+    div
+        [ class [ Style.MovieModal ]
+        ]
+        ([ button
+            [ class [ Style.CloseButton ]
+            , onClick (focusMovie NotSelected)
```

Comment:
Here's where we call `FocusMovie` (passed in through the modal view) to unselect/close.